armi/reactor/blocks.py

```diff
@@ -1812,6 +1840,10 @@ def getDuctOP(self):
         .. impl:: OP dimension is retrievable.
             :id: I_ARMI_BLOCK_DIMS4
             :implements: R_ARMI_BLOCK_DIMS
+
+            This method retrieves the duct Component and quieries
+            it's inner pitch directly. If the duct is missing or if there
+            are multiple duct Components, an error will be raised.
         """
```

Comment:
This method queries it's outer pitch.

Reply:
doh! good catch https://github.com/terrapower/armi/pull/1619/commits/d809a51cd344066be8d83cba789aefb09ffb378c